### ISSUE TITLE
[BUG] v1.4.1.0 unhandled exception on start

### ISSUE THREAD

**Issue by matthewhopkins** [Squirrelies/RE2REmakeSRT#22]
**Computer Information:**
 - OS: Windows 10 Version 1903 (OS Build 18362.535)
 - SRT version: 1.4.1.0
 - Game version: Steam latest

**Describe the bug**
On launch attempt an unhandled exception window comes up and the program doesn't start.

**Steps to Reproduce**
1. Launch RE2
2. Launch SRT

**Expected behavior**
The program to launch

**Screenshots**
![srt_1410_exception](https://user-images.githubusercontent.com/4790241/71756340-139e0b80-2e44-11ea-8a7e-65176dc35146.JPG)

**Additional context**
RE2 Windowed Mode and DX11

**Comment by Squirrelies** [comment]
Everything was extracted to the same folder? You should have roughly 59 files in your srt folder. Otherwise, there may be a new requirement for .NET Core 3.1 Runtime. I'd be surprised if that was the case but it is a possibility..

**Comment by matthewhopkins** [comment]
> 
> 
> Everything was extracted to the same folder? You should have roughly 59 files in your srt folder. Otherwise, there may be a new requirement for .NET Core 3.1 Runtime. I'd be surprised if that was the case but it is a possibility..

All files extracted to the same dir, will install .NET Core 3.1 and try again.

**Comment by matthewhopkins** [comment]
I tried explicitly installing .NET Core 3.1 x64+x86 and ASP.NET Core 2.2 Runtime x64+x86, same error dialogue.

Also additional context here is my dotnet info.

```
Microsoft Windows [Version 10.0.18362.535]
(c) 2019 Microsoft Corporation. All rights reserved.

C:\>dotnet --info
  It was not possible to find any installed .NET Core SDKs
  Did you mean to run .NET Core SDK commands? Install a .NET Core SDK from:
      https://aka.ms/dotnet-download

Host (useful for support):
  Version: 3.1.0
  Commit:  65f04fb6db

.NET Core SDKs installed:
  No SDKs were found.

.NET Core runtimes installed:
  Microsoft.AspNetCore.All 2.2.0 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.App 2.2.0 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 3.1.0 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.WindowsDesktop.App 3.1.0 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]

To install additional .NET Core runtimes or SDKs:
  https://aka.ms/dotnet-download
```

**Comment by matthewhopkins** [comment]
Also receive the same error dialogue on version 1.4.0.0

**Comment by Squirrelies** [comment]
One last question, did you download from the links in the ChangeLog.MD or from the Releases tab?

**Comment by Squirrelies** [comment]
I believe I found the issue. The Releases tab CI automated builds are not including RE2REmakeSRT.exe.config. As soon as I copied that into the extracted folder, it started working for that build. I'll work on fixing that (if I can remember which login I used for Azure DevOps >.>).

In the meantime, use the http://dudley.gg/squirrelies/re2/latest.7z or explicit http://dudley.gg/squirrelies/re2/RE2REmakeSRT-1410-Beta-Signed-Release.7z download.
